tests/test_config.py

```diff
@@ -973,6 +985,25 @@ def test_init__with_v4_datafile(self):
     self.assertEqual(expected_rollout_id_map, project_config.rollout_id_map)
     self.assertEqual(expected_variation_variable_usage_map, project_config.variation_variable_usage_map)
 
+  def test_variation_has_featureEnabled_false_if_prop_undefined(self):
+    """ Test that featureEnabled property by default is set to False, when not given in the data file"""
+    variation = {
```

Comment:
Please indent this correctly

Reply:
Do you mean 4 space indent? The whole file right now is on a 2 space indent.

Reply:
Hmm github just shows it weird. See how line 1003 (the closing bracket) does not match the indentation of the opening bracket on line 990